### ISSUE TITLE
Scourge Strike - Unholy DK

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#2195]
Originally reported on Google Code with ID 2196

```
IF RUNNING FROM THE GUI, PLEASE ATTACH (or Cut-N-Paste contents) OF
simc_gui.simc FILE.  THIS WILL ENABLE TO US TO REPRODUCE THE PROBLEM.

What steps will reproduce the problem?
1. Scourge Strike damage out for Unholy DK is too low in the sim.

What is the expected output? What do you see instead?

A few weeks ago Scourge Strike was 78% physical and 42% shadow.  It was then nerfed
5%, presumably, to 74% physical and 40% shadow.  

Scourge Strike was then buffed 50%.  

TL:DR: Scourge Strike should now be\111% physical and 60% shadow.



What version of the product are you using? On what operating system?

simc-603-5-win64-11-02-9c83806


Please provide any additional information below.


```

Reported by `jwm142011` on 2014-11-03 04:11:20
